src/AppBundle/Resources/views/Restaurant/index.html.twig

```diff
@@ -126,20 +126,34 @@
                   {{ option.name }}
                   {% if option.strategy == 'option' %} - {{ option.price|price_format }}€{% endif %}
                 </h4>
-                <div class="list-group" data-product-option>
+                <div class="list-group" {% if not option.additional %}data-product-option{% endif %}>
```

Comment:
I think you should add a text on the modal intended to the customer. Like "optional" next to the modifier's name.

Reply:
I think the option name can be used in this case, for example "Would you like an extra?"